modules/manage/partials/internal-use-property.adoc

```diff
@@ -0,0 +1 @@
+WARNING: This property is for Redpanda internal use only. Do not modify it unless specifically instructed by Redpanda Support.
```

Comment:
Can we link to [su](https://support.redpanda.com/hc/en-us)

Reply:
Yes! Sounds good to me